dbtmetabase/__main__.py

```diff
@@ -359,12 +359,20 @@ def models(
     is_flag=True,
     help="Include personal Metabase collections.",
 )
+@click.option(
+    "--exclude-unverified-cards",
```

Comment:
I think it should still be called `--exclude-unverified` in case dashboards ever become verified. "Cards" is also a term I avoid, because it doesn't appear anywhere in the UI (they're called questions), so Metabase users unfamiliar with its API won't know what it means

Reply:
> should still be called `--exclude-unverified` in case dashboards ever become verified

I was internally debating this before pushing the updated change. The issue with keeping the exclusion general to all model types is that it creates ambiguity for model types that *can't* be verified. For example, is a dashboard considered unverified? Or were you implying we should actually change the logic to exclude dashboards altogether when this flag is enabled?

> "Cards" is also a term I avoid [...] Metabase users unfamiliar with its API won't know what it means.

That's very true; I agree 💯 . Given the point I made above, I think we should call the flag `--exclude-unverified-questions`. Thoughts?

Reply:
No, I'm happy with only applying filtering to questions for now.

What I want to avoid is adding too many niche settings (considering nobody had requested this feature before), e.g. if/when dashboards can be verified, instead of adding yet another setting `--exclude-unverified-dashboards`, I'd rather just announce a behaviour change that this now filters dashboards too.

Reply:
Given what you're saying, I'm starting to think we want an `--include-verified-only` flag instead and strictly filter on anything that has the verification status. This seems like the only option that gives us both 1) clarity in current functionality and 2) future-proof API.

Reply:
But again, very happy to keep the current logic and revert the flag's name to `--exclude-unverified` if that's what you prefer. We can make it clear in the documentation what this flag does.

Reply:
"Exclude unverified" and "only include verified" are logically identical, I prefer the former because it's shorter. Neither stipulates how to handle unverifiable entities (e.g. dashboards), so we have to make a decision either way (i.e. we include unverifiable entities regardless).

Reply:
pushed the change back to `--exclude-unverified`. Let me know if you're good with the wording of the help docs.